sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQuerySqlDialect.java

```diff
@@ -253,6 +261,12 @@ private void unparseTrim(SqlWriter writer, SqlCall call, int leftPrec, int right
     writer.endFunCall(trimFrame);
   }
 
+  private void unparseDateTimeLiteralWrapperFunction(
+      SqlWriter writer, SqlCall call, int leftPrec, int rightPrec) {
+    writer.literal("DATETIME");
+    writer.literal(call.operand(0).toString().substring(DATETIME_LITERAL_OFFSET));
```

Comment:
I would recommend using `replace` (replace `TIMESTAMP` with `DATETIME`) here instead of `substring`. I think that's more readable.

Reply:
Make sense. Done.